src/codewars/m_sulkouski/github/com/TrafficLight.java

```diff
@@ -0,0 +1,139 @@
+package codewars.m_sulkouski.github.com;
+
+import javafx.scene.effect.Light;
+
+import java.util.LinkedList;
+
+public class TrafficLight {
+
+
+    public static void main(String[] args) {
+        String[] roadSituation = trafficLights("C...R............G......", 10);
```

Comment:
Always try to avoid hardcoded strings. C, R, and G should be variable constants:
final String RED = "RED";

If you want to go further you can even create a enumerator ;)